dev/src/setup/setup.ts

```diff
@@ -107,7 +108,7 @@ export async function setup() {
         const env = new Environment(pair)
         await env.isReady()
 
-        const dappStakeDefault = await env.contractInterface.getDappStakeDefault()
+        const dappStakeDefault = (await env.contractInterface.getDappStakeDefault()).muln(2)
```

Comment:
why is it multiplied by 2?

Reply:
Because the contract iteration in captca has a bug in it where it's looking for `> dapp_stake_default` in order to mark a dapp as active. This ensures that the dapp is active when they're registered.

Reply:
ah I see :)